tests/integration/targets/attachment/res/sample_file_changed.txt

```diff
@@ -0,0 +1,5 @@
+One morning, when Gregor Samsa woke from troubled dreams, he found himself transformed in his bed into a horrible vermin. He lay on his armour-like back, and if he lifted his head a little he could see his brown belly, slightly domed and divided by arches into stiff sections. The bedding was hardly able to cover it and seemed ready to slide off any moment. His many legs, pitifully thin compared with the size of the rest of him, waved about helplessly as he looked. "What's happened to me?" he thought. It wasn't a dream. His room, a proper human room although a little too small, lay peacefully between its four familiar walls. A collection of textile samples lay spread out on the table - Samsa was a travelling salesman - and above it there hung a picture that he had recently cut out of an illustrated magazine and housed in a nice, gilded frame. It showed a lady fitted out with a fur hat and fur boa who sat upright, raising a heavy fur muff that covered the whole of her lower arm towards the viewer. Gregor then turned to look out the window at the dull weather.
```

Comment:
Please generate a random string using `community.general.random_string` or `urandom` command.

Reply:
I must admit I did not look into this PR yet since it is still marked as a draft. But I agree that we should go with the usual lorem ipsum text here instead of using the literature excerpts.

Reply:
I did not plan to keep this in the final version, I only used it because it was easier for me to see whether the file changed or not. I used Lorem Ipsum in the "original" file, so I needed another distinct text to visually differentiate the files. Either way, this is still a draft, so I'm planning to do quite a few changes before it's ready for review.